pandevice/__init__.py

```diff
@@ -66,10 +66,17 @@ def getlogger(name=__name__):
 # Enumerator type
 def enum(*sequential, **named):
     enums = dict(zip(sequential, range(len(sequential))), **named)
-    reverse = dict((value, key) for key, value in enums.iteritems())
+    reverse = dict(((v, k) for (k, v) in enums.items()))
     enums['reverse_mapping'] = reverse
     return type('Enum', (), enums)
 
+def isstring(arg):
+    try:
+        return isinstance(arg, basestring)
+    except NameError:
+        return isinstance(arg, str) or isinstance(arg, bytes)
```

Comment:
Added a suggestion on another way to handle `element_str()`, since the `ElementTree.tostring()` method is capable of producing unicode natively in python 3.  See my comment here:  https://github.com/PaloAltoNetworks/pandevice/pull/59#discussion_r120527512